etl/steps/data/garden/ophi/2022-12-13/multidimensional_poverty_index.meta.yml

```diff
@@ -0,0 +1,359 @@
+
+          If the household survey data being used is missing any of the 10 indicators, that indicator is dropped from the calculation. The weights are then adjusted so that each dimension continues to be given a weight of one-third. MPI poverty estimates are only calculated if at least one indicator in each dimension is available.
+    - mpi_first:
+        description: &description-mpi_first |
+          The Multidimensional Poverty Index is obtained by multiplying two values: the share of people who are multidimensionally (MPI) poor and the intensity of multidimensional poverty among the MPI poor. A larger figure represents a higher level of poverty.
+    - mpi_last:
+        description: &description-mpi_last |
+          The Multidimensional Poverty Index, being the product of the incidence and intensity of multidimensional poverty, reflects both.
+    - intensity:
+        description: &description-intensity |
+          The intensity of multidimensional poverty is calculated as the average share of indicators in which those counted as MPI poor are deprived (using the same weights to calculate the average). This is an important complementary measure to the share of the population who are MPI poor (the 'incidence' of MPI poverty). An example given by the researchers who calculate the MPI data serves to illustrate this well:
+
+          'Imagine two countries: in both, 30 percent of people are poor (incidence). Judged by this piece of information, these two countries are equally poor. However, imagine that in one of the two countries poor people are deprived—on average—in one-third of the dimensions, whereas in the other country, the poor are deprived—on average—in two-thirds. By combining the two pieces of information - the intensity of deprivations and the proportion of poor people - we know that these two countries are not equally poor, but rather that the second is poorer than the first because the intensity of poverty is higher.'
+    - cme:
+        description: &description-cme |
+          This variable is a current margin estimate (CME), based on the most recent survey data. Look for the harmonized over time (HOT) estimate to see trends over time.
+    - hot:
+        description: &description-hot |
+          This variable is a harmonized over time (HOT) estimate. This harmonization seeks to make two or more MPI estimations comparable by aligning the indicator definitions in each survey. Look for the current margin estimate (CME) to see the most recent survey data.
+
+tables:
+  multidimensional_poverty_index:
+    variables:
+      cme_mpi_national:
+        title: MPI (national) - Current margin estimate
+        unit: ""
+        short_unit: ""
+        description:
+          - *description-cme
+          - "Multidimensional Poverty Index.\n"
```

Comment:
Have you checked on your local grapher or stating that these texts look good in grapher? I still see a bunch of line breaks like `\n`. I suppose they should be removed.

Reply:
They work fine. I added those because there was not line break between the text and the anchors.